prefect_twitter/credentials.py

```diff
@@ -1,49 +1,66 @@
 """Credential classes used to perform authenticated interactions with Twitter"""
 
 from dataclasses import dataclass
-from tkinter import E
-from typing import Optional, Union
 
-from tweepy import Client
+from tweepy import API, OAuth1UserHandler
+
 
 @dataclass
 class TweepyCredentials:
```

Comment:
Can we call this class `TwitterCredentials`? The fact that it uses `tweepy` under the hood is an implementation detail.
```suggestion
class TwitterCredentials:
```